src/Engine/Objects/PlayerEnums.h

```diff
@@ -277,26 +277,32 @@ enum class PLAYER_SKILL_TYPE : int8_t {
     PLAYER_SKILL_ALCHEMY = 35,
     PLAYER_SKILL_LEARNING = 36,
     PLAYER_SKILL_CLUB = 37, // In vanilla clubs are using separate hidden & non-upgradable skill.
-    PLAYER_SKILL_COUNT = 38,
+    PLAYER_SKILL_MISC = 38, // Hidden skill that's always 1. Used for wetsuits, for example.
+    PLAYER_SKILL_COUNT = 39,
 
-    PLAYER_SKILL_MISC = 38, // Hidden skill that's always zero, not stored anywhere
-
-    PLAYER_SKILL_MM7_FIRST = PLAYER_SKILL_STAFF,
-    PLAYER_SKILL_MM7_LAST = PLAYER_SKILL_LEARNING,
+    PLAYER_SKILL_FIRST_STORED = PLAYER_SKILL_STAFF,
```

Comment:
what STORED stands for? what stored and where?

Reply:
FIRST_VISIBLE will be a better name I think